script.js

```diff
@@ -0,0 +1,13 @@
+function calc() {
+
+    var a = parseInt(document.querySelector("#number1").value);
+    var b = parseInt(document.querySelector("#number2").value);
```

Comment:
While this works, we always emphasize in code naming things what they are. Naming these something like `num1` and `num2` reduces the mental mapping you have to do to remember `a` is from the `number1` field. Feels easy on a small code base like this, becomes much harder in large code bases with thousands of variables.